slurm-job-exporter.py

```diff
@@ -151,6 +151,16 @@ def __init__(self, dcgm_update_interval=10):
                         dcgm_fields.DCGM_FI_PROF_NVLINK_RX_BYTES: 'nvlink_rx_bytes',
                     }
 
+                    for gpu_id in pydcgm.DcgmSystemDiscovery(self.handle).GetAllSupportedGpuIds():
+                        device = pydcgm.dcgm_agent.dcgmGetDeviceAttributes(self.handle.handle, gpu_id)
+                        name = device.identifiers.deviceName
+                        print('Detected gpu {} with ID {}'.format(name, gpu_id))
+                        if name in ['Quadro RTX 6000']:
```

Comment:
The patch works with this change.

```suggestion
                        if name in ['NVIDIA RTX A6000']:
```